spec/lang/well-formed.md

```diff
@@ -76,24 +91,28 @@ impl Type {
                 for (offset, ty) in fields {
                     // Recursively check the field type.
                     ty.check_wf::<T>()?;
+                    // Ensure it is a sized type.
+                    ensure_wf(ty.size::<T>().is_sized(), "Type::Tuple: unsized field type")?;
                     // And ensure it fits after the one we previously checked.
                     ensure_wf(offset >= last_end, "Type::Tuple: overlapping fields")?;
-                    last_end = offset + ty.size::<T>();
+                    last_end = offset + ty.size::<T>().expect_sized("Ensured to be sized above");
```

Comment:
```suggestion
                    last_end = offset + ty.size::<T>().expect_sized("ensured to be sized above");
```
We follow rustc diagnostics style here (also applies a few more times in this PR)

Reply:
Would the acronym "WF" also be lowercased in the beginning of the sentence?

Reply:
"WF" is fine